client/src/components/EncryptCheckbox.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Label = styled.label`
+  margin: 10px 0px 5px 0px;
+`;
+
+const Checkbox = styled.input`
+  margin-left: 10px;
+`;
+
+export default function EncryptCheckbox({ value, onChange, label }) {
```

Comment:
This is similar to https://github.com/lmachens/copypasta/blob/master/client/src/components/Checkbox.js